core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -638,7 +638,7 @@ object DumpLogSegments {
       " Instead, the value-decoder-class option can be used if a custom RLMM implementation is configured.")
     private val shareStateOpt = parser.accepts("share-group-state-decoder", "If set, log data will be parsed as share group state data from the " +
       "__share_group_state topic.")
-    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Whether to skip printing metadata for each record.")
+    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Skip metadata when printing records. This flag also skips control records.")
```

Comment:
Its better then before, thank you.

Flag is kind of clear from the context.

Now its actually like the var name itself, summarizing is down. 

How you like this?

done need verb print(ing) and dont need 2 times skip when making an include to exclude/skip.


```suggestion
    private val skipRecordMetadataOpt = parser.accepts("skip-record-metadata", "Skip record metadata print. This also includes control records.")
```

Reply:
I'm not a native speaker, but the 2nd phrase could create confusion because one could read "skip AND include", as if the command weren't read-only.